irods_log.log

```diff
@@ -0,0 +1,161 @@
+[02/Jul/2024 14:18:30] INFO [irods_browser_app.views:114] ************************************************************************************************************************************************************************************************************************************************************************************************************
```

Comment:
This looks like it was accidentally included in a commit

Reply:
yep, thanks